consensus/src/runway/mod.rs

```diff
@@ -588,15 +589,26 @@ where
         }
     }
 
+    fn backup_data(
```

Comment:
I think runway should already be getting an abstracted unit saver, implemented by a wrapper applied by member to the `Write`. Same for `Read` ofc.

I also think this is not the PR to do this in, which is why I suggested adding `_` in front of the arguments – although I probably should have suggested this in member already.